### ISSUE TITLE
add_fromtable() is wrong for sw > 1-sorw

### ISSUE THREAD

**Issue by berland** [equinor/pyscal#50]
The following code illustrates a bug with `WaterOil.add_fromtable()`:
```
from pyscal import WaterOil
import matplotlib.pyplot as plt

(fig, ax) = plt.subplots()

wo = WaterOil(sorw=0.3, h=0.01)
wo.add_corey_water(nw=2, krwend=0.8)
wo.add_corey_oil(now=2)
wo.plotkrwkrow(ax=ax)

wo_fine = WaterOil(h=0.0001)
wo_fine.add_fromtable(wo.table, swcolname="sw")
wo_fine.plotkrwkrow(ax=ax, color="green")
plt.show()
```
![Screenshot from 2019-10-28 18-32-01](https://user-images.githubusercontent.com/306834/67702060-5d382900-f9b1-11e9-8792-ab2aa01bf7f2.png)

The green curve should follow the blue in this plot.

The reason is that add_fromtable() is using a monotone cubic interpolator, which makes a lot of sense for `sw < 1 - sorw`, but not above.

**Comment by berland** [comment]
When #49 is done, this can be used to separate where to apply cubic interpolation. If #49 is not done, we should not do cubic interpolation at all.

**Comment by berland** [comment]
Potentially solved in #51

**Comment by berland** [comment]
Solved in #51
